notebooks/236-stable-diffusion-v2/236-stable-diffusion-v2-text-to-image.ipynb

```diff
@@ -2,10 +2,8 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    int8_unet_model = core.compile_model(UNET_INT8_OV_PATH, device.value)
</pre><p>It is worth trying dynamic quantization of activations available in the latest OpenVINO version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
@AlexKoff88 , I've tried dynamic quantization by setting `core.set_property("CPU", properties={"DYNAMIC_QUANTIZATION_GROUP_SIZE":"32"})` and keeping everything else as is, but got `1.13x` performance boost instead of previous `1.23x`.

Reply:
@l-bat used 4-b-bit weight compression from what I know. It should lead to better performance when stacking with dynamic quantization.

Reply:
@AlexKoff88 With `INT4_ASYM, group_size=64, ratio=1.0` the perf. boost improves performance to `1.18x`

Reply:
Thanks @nikita-savelyevv for the data. They are very helpful. 

@dmitry-gorokhov, fyi